src/frontend/komponenter/Felleskomponenter/HeaderMedSøk/FagsakDeltagerskort.tsx

```diff
@@ -0,0 +1,62 @@
+import './fagsakdeltagerskort.less';
+
+import React from 'react';
+
+import { Infokort } from '@navikt/familie-header';
+import {
+    GuttIkon, JenteIkon, KvinneIkon, MannIkon, NøytralPersonIkon
+} from '@navikt/familie-ikoner';
+import { kjønnType } from '@navikt/familie-typer';
+
+import { FagsakDeltagerRolle } from '../../../typer/fagsakdeltager';
+
+export interface IFagsakDeltagerskortProps {
+    navn: string;
+    ident: string;
+    rolle: FagsakDeltagerRolle;
+    index: number;
+    kjønn?: kjønnType;
+    fagsakId?: string;
+    onClick?: (index: number) => void;
+    children?: React.ReactNode | React.ReactNode[];
+}
+
+const FagsakDeltagerskort: React.FunctionComponent<IFagsakDeltagerskortProps> = ({
+    navn, ident, rolle, index, kjønn, fagsakId, onClick, children
+}) => {
+    type IkonerMap = Record<string, React.ReactNode>;
+    type RolleNavnMap = Record<string, string>;
+
+    const ikoner: IkonerMap = {
```

Comment:
Why not use FamilieIkonVelger?

Reply:
To my understanding, the icon here is based on the role of the person, not the age. For example, if a person is 17 years old, but he/she is an applicant of a case, we should display he/she as a parent, not a child.

Reply:
Right! 🙌